toolchain/check/import_cpp.cpp

```diff
@@ -187,6 +188,43 @@ static auto ClangLookup(Context& context, SemIR::LocId loc_id,
   return lookup;
 }
 
+// Returns the return type of the given function declaration.
+// Currently only void and 32-bit int are supported.
+// TODO: Support more return types.
+static auto GetReturnType(Context& context,
+                          const clang::FunctionDecl* clang_decl)
+    -> SemIR::InstId {
+  auto ret_type = clang_decl->getReturnType().getCanonicalType();
+  if (ret_type->isVoidType()) {
+    return SemIR::InstId::None;
+  }
+  const int supported_int_width = 32;
+  if (ret_type->isSignedIntegerType() &&
+      clang_decl->getASTContext().getTypeSize(ret_type) ==
+          supported_int_width &&
+      clang_decl->getASTContext().getTargetInfo().getIntWidth() ==
+          supported_int_width) {
+    auto size_id = context.ints().Add(supported_int_width);
+    auto type_id = MakeIntType(context, Parse::NodeId::None,
+                               SemIR::IntKind::Signed, size_id);
+    auto type_inst_id = MakeIntTypeLiteral(context, Parse::NodeId::None,
+                                           SemIR::IntKind::Signed, size_id);
+
+    auto return_slot_pattern_id = AddInstInNoBlock(
+        // TODO: have a location for the return type once available.
```

Comment:
```suggestion
        // TODO: Have a location for the return type once available.
```

Style: per https://google.github.io/styleguide/cppguide.html#TODO_Comments, we try to capitalize the first word of a TODO.

Reply:
Done, thanks.